### PR TITLE
Ensure chain notifications send a full label in the email

### LINKED ISSUE #895: Ensure chain notifications send a full label in the email
Currently they only display the title, e.g. "Bonded".

They should have the full label in the body of the notification, along with a link to Commonwealth to the notifications list (scrolled down to that notification).

### PR DESCRIPTION
**Pull request by zakhap**
## Description

- Add whole subject line to email body instead of just the "bonded"/event label. 
- Also create a new link for chain events to go to the notificationsList and scroll down to the specific notification.
Closes #895.

## Motivation and Context

More info for the user and better UX for the link back to CW.

## How has this been tested?

I only tested on a single chain event.

## Have proper tags been added (for bug, enhancement, breaking change)?
- [x] yes

## Does this PR affect any server routes?
- [x] no

### CODE REVIEW

File: server/scripts/emails.ts [hunk 1]
```diff
@@ -51,8 +51,8 @@ export const createImmediateNotificationEmailObject = async (notification_data,
         notification: {
           chainId: notification_data.chainEventType?.chain,
           blockNumber: notification_data.chainEvent?.blockNumber,
-          label: chainEventLabel.heading,
-          path: `https://commonwealth.im${chainEventLabel.linkUrl}`,
+          label: subject,
```
Comment (dillchen) [inline server/scripts/emails.ts:54]: can we make sure that the chain in the subject is capitalized?

Comment (dillchen) [inline server/scripts/emails.ts:54]: can we make sure that the chain in the subject is capitalized?

File: server/models/subscription.ts [hunk 2]
```diff
@@ -173,6 +173,9 @@ export default (
             notification_data: JSON.stringify(notification_data)
           }
       );
+      if (msg && isChainEventData(notification_data)) {
+        msg.dynamic_template_data.notification.path = `https://commonwealth.im/${notification_data.chainEventType.chain}/notificationsList?id=${notification.id}`;
```
Comment (dillchen) [inline server/models/subscription.ts:177]: Can you update this to ${SERVER_URL} instead of commonwealth.im. And then I think we should be all good

Comment (zakhap) [reply server/models/subscription.ts]: resolved

**Pull request merged**